lib/countdown/ui/countdown_section.dart

```diff
@@ -233,7 +227,7 @@ class _LightButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) => GenericButton(
         title: title,
-        onPressed: onPressed,
+        onPressed: onPressed(),
```

Comment:
Why did we call the onPressed method? Have you tried this after the changes and did it work right?

Reply:
Because, it had error.

<img width="539" alt="Ekran Resmi 2022-02-27 00 15 00" src="https://user-images.githubusercontent.com/13748518/155859315-df41ca24-c589-46be-98c4-c90dd1dc11c2.png">

Reply:
Interesting. Have you tried it is working correctly?